### ISSUE TITLE
Add title to interactive iframe

### ISSUE THREAD

**Issue by atsansone** [flutter/website#10530]
From: https://buganizer.corp.google.com/issues/337851628

### OBSERVED RESULTS
Navigating through the interactive code example, I noticed that the iframe is not having a defined title.

* [Screencast VoiceOver](http://screencast/cast/NjcwNzcxNDUxOTQ2NTk4NHxkN2JiNDY2YS0xYg)
* [Screencast ChromeVox](http://screencast/cast/NjcwMzg1NTEyMjI1MTc3NnxkNDY3ZGVjZC00ZQ)

### EXPECTED RESULTS
* Ensure that when the user is on the code section the `iframe` should have a definite name like: `Interactive code iframe text section`.

* Element has no title attribute aria-label attribute does not exist or is empty aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty Element's default semantics were not overridden with role="none" or role="presentation"

### USER IMPACT
Screen reader and Cognitive users are confused because they do not know what is the name of that section is.

### STEPS TO REPRODUCE

1. Enabled VoiceOver using "Command+F5" also ChromeVox using "Ctrl+Alt+Z"
2. Open the following page:[URL](https://docs.flutter.dev/cookbook/design/snackbars#interactive-example)
3. Continue the linear navigation using the "Tab" and "Arrow" keys and reach the interactive example code.
4. Listen to announced the title on that iframe.

### GAR Assessment
[Flutter Docs](https://a11ytracker.corp.google.com/products/c2fdcf18-53d2-475d-b8b0-e5dedf8cb900?e=ProductLabels)

### GAR DETAILS
[1.1. Do all non-text content and user interface components have meaningful accessible names and alternative text?](http://go/GAR-web#accessible-names)

### ADDITIONAL INFORMATION
NA

### ENVIRONMENT DETAILS

- Machine: MAC/ChromeBook
- Browser: Chrome Version 120.0.6099.272 (Official Build) (64-bit)
- Screen reader:VoiceOver/ChromeVox.
- Bug filling time: 15 mins.
